src/Elyon.Fastly.Api.DomainServices/InfoSessionFollowUpService.cs

```diff
@@ -103,14 +104,27 @@ await _emailSenderService.SendInfoSessionFollowUpEmailAsync(receiver, specDto.Me
 
         private async Task SendCompanyOnboardingEmailAsync(InfoSessionFollowUpSpecDto specDto, OrganizationDto organization)
         {
+            var receiversCount = specDto.Receivers.ToList().Count;
+            var currentCount = 0;
             foreach (var receiver in specDto.Receivers)
             {
+                var ccReceivers = new List<string>();
+                if (receiversCount > 1 && currentCount == 0)
+                {
+                    ccReceivers = null;
+                }
+                else
+                {
+                    ccReceivers = specDto.CcReceivers.ToList();
```

Comment:
don't call tolist() every time. You've already called it in the first line of this method. Set it in a variable and use it whenever needed.